chapters/fr/chapter1/1.mdx

```diff
@@ -0,0 +1,51 @@
+- Les chapitres 9 à 12 proposent d'aller plus loin et d'explorer comment les modèles Transformer peuvent être utilisés pour résoudre des problèmes de traitement de la parole et de la vision par ordinateur. En suivant ces chapitres, vous apprendrez à construire et à partager vos modèles via des démonstrateurs, et vous serez capable d'optimiser ces modèles pour les environnements de production. Enfin, vous serez prêt à appliquer 🤗 Transformers à (presque) n'importe quel problème de machine learning!
+
+Ce cours:
+
+* Requiert un bon niveau en Python
+* Se comprend mieux si vous avez déjà suivi un cours d'introduction au deep learning, comme [fast.ai's](https://www.fast.ai/) [Practical Deep Learning for Coders](https://course.fast.ai/) ou un des cours développés par [DeepLearning.AI](https://www.deeplearning.ai/)
+* N'attend pas une connaissance appronfondie de [PyTorch](https://pytorch.org/) ou de [TensorFlow](https://www.tensorflow.org/), bien qu'être familiarisé avec l'un d'entre eux peut aider
+
+Après avoir terminé ce cours, nous vous recommandons de suivre la [Spécialisation en NLP](https://www.coursera.org/specializations/natural-language-processing?utm_source=deeplearning-ai&utm_medium=institutions&utm_campaign=20211011-nlp-2-hugging_face-page-nlp-refresh) dispensée par DeepLearning.AI, qui couvre une grande gamme de modèles traditionnels de NLP comme les Naïves Bayes et les LSTMs qui sont importants à connaître!
+
+## Qui sommes-nous ?
+
+À propos des auteurs de ce cours:
+
+**Matthew Carrigan** est un ingénieur en Machine Learning chez Hugging Face. Il vit à Dublin, en Irlande et à travaillé auparavant comme ingénieur en ML chez Parse.ly et avant cela comme chercheur postdoctoral à Trinity College Dublin. Il ne croit pas que nous arrivions à AGI en mettant à l'échelle les architectures existantes, mais il a tout de même beaucoup d'espoir dans l'immortalité des robots.
+
+**Lysandre Debut** est un ingénieur en Machine Learning chez Hugging Face et a travaillé sur la librairie 🤗 Transformers depuis les premières phases de développement. Son but est de rendre NLP accessible pour tout le monde en développant des outils disposant d'une API très simple.
+
+**Sylvain Gugger** est un ingénieur en recherche chez Hugging Face et un des principaux responsable de la librairie 🤗 Transformers. Précédemment, Il a été chercheur en ML chez fast.ai et a écrit _[Deep Learning for Coders with fastai and PyTorch](https://learning.oreilly.com/library/view/deep-learning-for/9781492045519/)_ avec Jeremy Howard. Son but ultime est de rendre le deep learning plus accessible, en développant et en améliorant des techniques permettant aux modèles d'apprendre rapidement sur des ressources limitées.
+
+**Merve Noyan** est développeuse advocate chez Hugging Face et travaille à la création d'outils et de contenu visant à démocratiser le machine learning pour tous.
```

Comment:
I'm wondering if job roles should be translated or not because `ingénieur en machine learning` means having a particular diploma from Engeenering school in France. I don't know if it's totally accurate.

Plus `développeuse advocate` looks really trash to me, is there any equivalent ? 

So should I keep english job titles ?

Reply:
Hey @ChainYo this is an interesting point - I think it's fine to use the English titles here if you want. If someone complains, we can always use the official French titles :)